src/liblsquic/lsquic_parse_gquic_Q041.c

```diff
@@ -443,11 +447,13 @@ gquic_ietf_gen_ack_frame (unsigned char *outbuf, size_t outbuf_sz,
     p += largest_acked_len;
 
     CHECKOUT(2);
-    lsquic_time_t diff = now - rechist_largest_recv(rechist);
-    gquic_be_write_float_time16(diff, p);
-    LSQ_DEBUG("%s: diff: %"PRIu64"; encoded: 0x%04X", __func__, diff,
-                                                            *(uint16_t*)p);
-    p += 2;
+    {
+        lsquic_time_t diff = now - rechist_largest_recv(rechist);
```

Comment:
Move `diff` declaration to top of function?

Reply:
I don't think that will work.  diff is used in a lot of places, scoped differently and it would be confusing.  Plus the line above what is shown here is a bswap_64 for Windows only.  Actually, I don't like using bswap_64 for Windows as it assumes 64-bit.  I might recommend a macro to do swapping if necessary - makes the code more readable.